samples/drivers/spi_fujitsu_fram/src/main.c

```diff
@@ -58,6 +58,7 @@ static int mb85rs64v_access(struct device *spi, struct spi_config *spi_cfg,
 		}
 	} else {
 		tx.count = 1;
+		bufs[0].len = 1;
```

Comment:
Please rewrite the commit title to "samples: spi_fujistu_fram: Fix buffer length"
Break some lines on the message, it should not be > 74 (or 72? can't recall) characters per line.

And add you Signed-off-by as well.